pkg/tsdb/azuremonitor/credentials.go

```diff
@@ -73,8 +74,6 @@ func normalizeAzureCloud(cloudName string) (string, error) {
 		return azsettings.AzureChina, nil
 	case azureMonitorUSGovernment:
 		return azsettings.AzureUSGovernment, nil
-	case azureMonitorGermany:
-		return azsettings.AzureGermany, nil
 	case azureMonitorCustomized:
 		return azsettings.AzureCustomized, nil
```

Comment:
You cloud just write directly here, do defined a local const for the Azure Monitor:

```go
return "AzureCustomizedCloud", nil
```
It would have the same effect but without defining it in SDK.

It's not going to work anyways.
Token provided won't be able to resolve authority:
https://github.com/grafana/grafana-azure-sdk-go/blob/3ba629ee4f48e9ded325bb3d5e34bf5649e79237/aztokenprovider/token_provider.go#L125

As mentioned before, it could work only if custom authority being passed with credential. In this case value of cloud would be irrelevant and could be any value, including `AzureCustomizedCloud`.

Reply:
leting this as it was before to not add more changes